### ISSUE TITLE
Documentation is not enjoyable to read

### ISSUE THREAD

**Issue by cthoyt** [mapping-commons/sssom#254]
Despite the advantages of using LinkML to define the SSSOM datamodel and autogenerate this page https://mapping-commons.github.io/sssom, I do not enjoy reading this page. 

As an developer who wants to produce SSSOM, I would like a table that says what are the columns I need to / could put in my TSV file and explanations on what needs each field is. I do not want to get this conflated with stuff that should appear constant in every row and is more logically a property of the entire file itself, like "license" and "documentation".

Somehow this should either be the first page in the standard, or there should be a massive, impossible-to-miss link pointing to such a table

**Comment by graybeal** [comment]
I agree, though I think this is a fine approach/document for the specification details.

I'm thinking a thoughtfully formatted example (with overlaid commentary) would be a better starting point than dropping someone directly into the standard. 

And a brief description (overview) before that of what they will be seeing would be even more helpful.

**Comment by matentzn** [comment]
I think you are totally right with everything you say, and it these pages are entirely customisable using jinja templates. 

With the current resources I cannot prioritise this right this moment, but we will do it before the Padua workshop during a major revision of the docs.

**Comment by udp** [comment]
I went to the docs trying to find out what I could put in a mapping field (specifically I wanted to know if an IRI was ok or whether it always had to be a CURIE like in the examples).

I scrolled down the page. Thankfully I'm reasonably familiar with LinkML so I know that what the docs call a "slot" corresponds to a column. So I looked in the slots for "subject_id", but they were arranged alphabetically so I had to do some scrolling through all the other random optional slots (there's nothing to indicate what's optional or otherwise from the table).

Finally I found subject_id, and I see this:

<img width="636" alt="Screenshot 2023-05-12 at 23 46 30" src="https://github.com/mapping-commons/sssom/assets/454895/d34052d0-9d7d-47ac-957e-93ea9ee4b8fc">

Ok, so can I put an IRI in this field? I click "EntityReference" which is apparently the range.

<img width="796" alt="Screenshot 2023-05-12 at 23 47 10" src="https://github.com/mapping-commons/sssom/assets/454895/d7f6a37a-68b2-4a40-a8f7-3c2e289d1f86">

I still (as a consumer of the docs who just wants to create a TSV file) have absolutely no idea what I can put in this field. It looks like any string or maybe a URI because rdf:Resource?

**Comment by graybeal** [comment]
Yes, let me clarify my earlier statement.

I think using auto-generation to produce documentation can work very well. This documentation could be more tuned to the naive user.

**Comment by graybeal** [comment]
Referencing #435 which was written without recall of this ticket...
